src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -585,6 +591,9 @@ private void checkAuthorizedUpdateUser(User user, DataAccessRequest dar) {
     if (!user.getUserId().equals(dar.getUserId())) {
       throw new ForbiddenException("User not authorized to update this Data Access Request");
     }
+    if (Objects.isNull(user.getLibraryCards()) || user.getLibraryCards().isEmpty()) {
```

Comment:
Out of curiosity, why `Objects.isNull` rather than `== null`?

Reply:
Consistency.  Since we do a good bit of `Collection Stream` processing, it seems like we've standardized on this instead of == null even in `if` conditions.  They are effectively the same.  See: [https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html)

Reply:
We've tried to move away from this more recently where we are not using streams, but this is a usage that is repeated throughout the codebase.

Reply:
One thing we can do to simplify the check is to instantiate `User.libraryCards` as an empty list so it's never null. We've been doing that in other classes recently (see [Study](https://github.com/DataBiosphere/consent/blob/f0f02ea7a573021f8358fd1b4d8009c675f57974/src/main/java/org/broadinstitute/consent/http/models/Study.java#L17) for example)

Reply:
We have an unexpected number of tests that fail when I return an empty list.  I can spend time on that now or create a ticket.  Which does the team prefer?

Reply:
Got to the bottom of the issues.  User.getLibraryCards() will now always return a list.